replica/server.go

```diff
@@ -376,6 +395,18 @@ func (s *Server) Close(signalMonitor bool) error {
 		return nil
 	}
 
+	if s.PreloadStatus != types.Done {
```

Comment:
why do we need a 'if' and then 'for' when you are however doing a 'log' in both the cases?

Reply:
I had used it for debugging purpose, removing it.